### ISSUE TITLE
Support parsing static methods

### ISSUE THREAD

**Issue by chase-ok** [duchess-rs/duchess#13]
It looks like our javap parser is failing after an unexpected `static` token before the return type for static methods:

```
error: unexpected token `...         public static  <<< int >>>  test();
                   d ...` at offset 1302, expected one of `["\"(\"", "\")\"", "\",\"", "\".\"", "\";\"", "\"<\"", "\">\"", "\"[\"", "\"extends\"", "\"implements\"", "\"{\"", "r#\"[a-zA-Z][a-zA-Z0-9]*\"#"]`
```

**Comment by nikomatsakis** [comment]
Huh, really? I think this must be fixed by #25

**Comment by chase-ok** [comment]
Confirmed, working on `main` now!
